common/src/main/java/tachyon/heartbeat/HeartbeatScheduler.java

```diff
@@ -63,19 +66,23 @@ public static void addTimer(ScheduledTimer timer) {
   public static synchronized void removeTimer(ScheduledTimer timer) {
     Preconditions.checkNotNull(timer);
     sLock.lock();
-    sTimers.remove(timer.getThreadName());
-    sLock.unlock();
+    try {
+      sTimers.remove(timer.getThreadName());
+    } finally {
+      sLock.unlock();
+    }
```

Comment:
Sure, there's no arguing about the correctness, but it would be good to make the style consistent (ie. always use `try..catch` regardless of if subsequent methods can fail).